lib/ts/controllers/s-navigation-tablist.ts

```diff
@@ -0,0 +1,121 @@
+namespace Stacks {
+    export class TabListController extends Stacks.StacksController {
```

Comment:
nit: I'd even just call this `TabController` since we could use it for lots of different things. Pagination comes to mind immediately and we could extend this in the future (like we did for Popovers>Tooltips) to handle collapsible and others.

Reply:
I prefer `TabList` here because it is intended to function on a `TabList` element and makes assumptions / adheres to the prescribed behavior.  I could imagine doing a similar controller for pagination but it would likely function very differently and have very different niche considerations.